Some API tests are failing - need to investigate prior to merge

```
=========================== short test summary info ============================
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_select[ds6] - assert 400 == 200
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_select[ds7] - assert 400 == 200
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_select[ds126] - assert 500 == 200
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_select[ds127] - assert 500 == 200
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_select[ds130] - assert 500 == 200
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_select[ds131] - assert 500 == 200
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_select[ds136] - assert 500 == 200
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_select[ds137] - assert 500 == 200
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_select[ds138] - assert 500 == 200
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_select[ds139] - assert 500 == 200
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_select[ds144] - assert 500 == 200
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_select[ds145] - assert 500 == 200
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_select[ds146] - assert 500 == 200
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_select[ds147] - assert 500 == 200
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_stats[ds6] - assert 400 == 200
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_stats[ds7] - assert 400 == 200
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_stats[ds128] - ValueError: Some fields are currently unavailable: {'ang_x'}. Please remove...
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_stats[ds129] - ValueError: Some fields are currently unavailable: {'ang_x'}. Please remove...
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_stats[ds132] - ValueError: Some fields are currently unavailable: {'ang_y'}. Please remove...
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_stats[ds133] - ValueError: Some fields are currently unavailable: {'ang_y'}. Please remove...
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_stats[ds134] - ValueError: Some fields are currently unavailable: {'ang_z'}. Please remove...
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_stats[ds135] - ValueError: Some fields are currently unavailable: {'ang_z'}. Please remove...
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_stats[ds140] - ValueError: Some fields are currently unavailable: {'lin_x'}. Please remove...
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_stats[ds141] - ValueError: Some fields are currently unavailable: {'lin_x'}. Please remove...
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_stats[ds142] - ValueError: Some fields are currently unavailable: {'lin_y'}. Please remove...
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_stats[ds143] - ValueError: Some fields are currently unavailable: {'lin_y'}. Please remove...
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_stats[ds148] - ValueError: Some fields are currently unavailable: {'lin_z'}. Please remove...
FAILED src/masschange/api/tests/test_api_ops.py::test_gracefo_data_stats[ds149] - ValueError: Some fields are currently unavailable: {'lin_z'}. Please remove...

```

--- a/pytest.ini
+++ b/pytest.ini
@@ -1,0 +1,5 @@
+[pytest]
+
+log_cli = true
+log_cli_level = ERROR
+log_cli_format = %(asctime)s [%(levelname)s] %(message)s

--- a/src/masschange/api/tests/test_api_ops.py
+++ b/src/masschange/api/tests/test_api_ops.py
@@ -47,7 +47,7 @@ channel_id_additional_parameters = {
 }
 
 
-@pytest.mark.parametrize("ds", permute_all_datasets())
+@pytest.mark.parametrize("ds", permute_all_datasets(), ids=lambda ds: ds.get_table_name())
 def test_gracefo_data_select(ds: Dataset):
     data_span = ds.get_data_span()
     test_span_begin = data_span.begin if data_span is not None else datetime(2000, 1, 1)
@@ -65,7 +65,9 @@ def test_gracefo_data_select(ds: Dataset):
     content = response.json()
 
     if response.status_code != 200:
-        print(json.dumps(content))
+        # provide descriptions for details hidden by parametrization
+        logging.error(f'Non-200 response for dataset {ds.product.get_full_id()} version {ds.version} instrument {ds.instrument_id} at path {path}')
+        logging.error(f'{content=}')
     assert response.status_code == 200
 
     # Omit variable-data-span-datasets from the test as the expected data count is unknown
@@ -127,7 +129,10 @@ def test_gracefo_data_stats(ds: TimeSeriesDataset):
         content = response.json()
 
         if response.status_code != 200:
-            print(json.dumps(content))
+            # provide descriptions for details hidden by parametrization
+            logging.error(
+                f'Non-200 response for dataset {ds.product.get_full_id()} version {ds.version} instrument {ds.instrument_id} field {field.name} at path {path}')
+            logging.error(f'{content=}')
         assert response.status_code == 200
 
         expected_attributes = ['from_isotimestamp', 'to_isotimestamp', 'field', 'statistic', 'result', 'query_elapsed_ms']
@@ -219,7 +224,7 @@ def test_product_metadata_basic():
                 assert agg['type'] in recognised_aggregation_types
 
 
-@pytest.mark.parametrize("ds", permute_all_datasets())
+@pytest.mark.parametrize("ds", permute_all_datasets(), ids=lambda ds: ds.get_table_name())
 def test_dataset_metadata(ds: TimeSeriesDataset):
     path = f'/missions/{ds.product.mission.id}/products/{ds.product.id_suffix}/versions/{ds.version}/instruments/{ds.instrument_id}'
     response = client.get(path)

